ui/src/pages/overview/OverviewGraph.tsx

```diff
@@ -124,6 +128,18 @@ export const OverviewGraph: React.FC = () => {
     }
   }, [data?.overviewPage.graph.attributes, onTelemetryTypeChange]);
 
+  const reactFlowWidth = useStore((state: { width: any }) => state.width);
+  const reactFlowHeight = useStore((state: { height: any }) => state.height);
+  const reactFlowNodeCount = useStore(
+    (state: { nodeInternals: any }) =>
+      Array.from(state.nodeInternals.values()).length || 0
+  );
+  useEffect(() => {
+    setTimeout(() => {
+      reactFlowInstance.fitView();
+    }, 10);
```

Comment:
I'm curious if this still works with `0`.

```suggestion
    setTimeout(() => {
      reactFlowInstance.fitView();
    }, 0);
```

https://stackoverflow.com/questions/779379/why-is-settimeoutfn-0-sometimes-useful